python/lsst/jointcal/ccdImage.cc

```diff
@@ -66,13 +65,11 @@ void declareCcdImageList(py::module &mod) {
 
     cls.def("sizeValidForFit", &CcdImageList::sizeValidForFit);
 
-    // Make this behave like a std::list, which it is, but we can't wrap children of it easily.
+    // Make this behave like a std::list (which it is but we can't wrap std::list subclasses trivially)
```

Comment:
Can't we **make** jointcal use a regular STL container (probably preferably `vector`) somehow?